examples/sagemaker-tensorflow/sagemaker-merlin-tensorflow.ipynb

```diff
@@ -0,0 +1,1200 @@
+{
```

Comment:
<p><code>you probably want to execute the script outside the container. </code> where exactly? can one just build this container on their terminal or they need a specific env to build the container in?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/692/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>This part is confusing and we want to eventually remove it in https://github.com/NVIDIA-Merlin/Merlin/pull/707</p>